content/en/blog/interoperability/index.md

```diff
@@ -0,0 +1,76 @@
+
+At first, there were two competing test suites with different approaches to execute the tests. [Aries Protocol Test Suite (APTS)](https://github.com/hyperledger/aries-protocol-test-suite) includes an agent implementation that interacts with the tested agent through the protocol messages. On the other hand, [Aries Agent Test Harness (AATH)](https://github.com/hyperledger/aries-agent-test-harness) runs the tests operating the agent-client interface. This approach makes it possible to measure the compatibility of any two agent implementations. AATH seems to be the winning horse of the test suite race. Its test suite includes several test cases and has extensive reporting in place.
+
+### Aries Agent Test Harness
+
+Aries Agent Test Harness provides a BDD (behavioral driven) test execution engine and a set of tests derived from Aries RFCs. The aim is to run these tests regularly between different Aries agents (and agent frameworks) to monitor the compatibility score for each combination and catch compatibility issues.
+
+Harness operates the agents under test through backchannels. Backchannel is a REST interface defined by [an OpenAPI definition](https://github.com/hyperledger/aries-agent-test-harness/blob/main/docs/assets/openapi-spec.yml), and its purpose is to pass the harness requests to the agents. The target is to handle the agent as a black box without interfering with the agent's internal structures. Thus, the backchannel uses the agent's client interface to pass on the harness requests.
+
+{{< figure src="https://courses.edx.org/assets/courseware/v1/571727dd6d3f57d64158c9567f0d8ff2/asset-v1:LinuxFoundationX+LFS173x+1T2020+type@asset+block/The_Aries_Agent_Test_Harness.png" attr="image source: LinuxFoundationX LFS173x (CC BY 4.0)" attrlink="https://learning.edx.org/course/course-v1:LinuxFoundationX+LFS173x+1T2020/home" >}}
+
+Harness utilizes Docker containers for testing. It launches a container based on a required agent image for each test scenario actor during the test run. Before the test run, one needs to build and bundle the agent-required services and the backchannel to a single agent image that provides both the agent and the backchannel functionality. The recipes for making each of the different agent images, i.e., Dockerfiles with the needed scripts, are stored in the AATH repository. The same repository also contains CI scripts for executing the tests regularly and generating [an extensive test report site](https://aries-interop.info/).
```

Comment:
"Before the test run, one needs to build and bundle the agent-required services and the backchannel to a single agent image that provides both the agent and the backchannel functionality. "
Maybe something like:
Before the test run, one needs to build a single image containing all the required agent services including the backchannel.